qa/pull-tester/rpc-tests.sh

```diff
@@ -161,6 +161,7 @@ testScripts=(
   'sc_getcertmaturityinfo.py',68,219
   'sc_big_commitment_tree.py',63,110
   'sc_big_commitment_tree_getblockmerkleroot.py',11,25
+  'p2p_ignore_spent_tx.py',1,1
```

Comment:
It could be worth to have a couple of runs on Travis in order to have an estimation of the execution times and update them accordingly.

Reply:
Definitely, this was just a placeholder for the first run of the CI. Updated with actual measurements.